Helper/Requests.php

```diff
@@ -367,6 +367,30 @@ public function buildRecurringData($request = [], $areaCode, int $storeId, $addi
         return $request;
     }
 
+    /**
+     * @param $request
+     * @param $areaCode
+     * @param $storeId
+     * @param $payment
+     */
+    public function buildRecurringAlternativePaymentMethodsData($request = [], $areaCode, int $storeId, $additionalData)
```

Comment:
```suggestion
    public function buildRecurringAlternativePaymentMethodsData($request = [], $areaCode, int $storeId)
```
not used, right?

Reply:
Right. Removed it from the call and definition.